doc/sphinx/source/input.rst

```diff
@@ -30,9 +29,15 @@ This section provides some guidelines for unfamiliar users.
     are most welcome. We are particularly interested in contributions from
     people with good understanding of the CORDEX project and its standards.
 
+This section provides an introduction to getting (access to) climate data
+for use with the ESMValTool.
```

Comment:
I've been told in the past that the tool should be referred to as `ESMValTool` rather than `the ESMValTool`

Reply:
yes good point! I've heard many people referring to it as "the ESMValTool" - I guess e can just let it pass but I agree we should refer to it simply as "ESMValTool" in the official docs

Reply:
Thanks, updated now